### PR TITLE
Bug 1121974 - Fix intermittent failure in test_settings_wifi.py, improve...

### PR DESCRIPTION
**Pull request by viorelaioia-zz**
... wait before typing the password

### CODE REVIEW

**Comment by try-server-hook** [comment]
[viorelaioia viorelaioia](https://github.com/viorelaioia) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=da7fd92178c1)

**Comment by try-server-hook** [comment]
[viorelaioia viorelaioia](https://github.com/viorelaioia) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=549745029af8)

**Comment by viorelaioia** [comment]
Adhoc run: http://jenkins1.qa.scl3.mozilla.com/job/flame-kk.ui.adhoc/594/console

File: tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/wifi.py [hunk 1]
```diff
@@ -5,10 +5,12 @@
 
 try:
     from marionette import Wait
+    from marionette import expected
```
Comment (bebef1987) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/wifi.py]: Please update this to fallow the new import pattern.

https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/apps/phone/app.py#L6

**Comment by try-server-hook** [comment]
[viorelaioia viorelaioia](https://github.com/viorelaioia) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ee24bbe77a00)

**Comment by try-server-hook** [comment]
[viorelaioia viorelaioia](https://github.com/viorelaioia) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=91bfab9efe02)

**Pull request merged**